ballerina/build.gradle

```diff
@@ -1082,6 +1082,11 @@ task testDevToolsIntegration() {
 
 task generateCache(dependsOn: ':cache-generator:build') {
     doFirst {
+        download {
+            src "https://raw.githubusercontent.com/ballerina-platform/ballerina-standard-library/main/release/resources/stdlib_modules.json"
```

Comment:
Shall we move `main` branch value as a Gradle property?
By default we can keep the `main` value, if any case we need to download a modified json file a different branch we can simply change the property. This can be happened during parallel releases